src/test/java/io/openliberty/tools/intellij/it/TestUtils.java

```diff
@@ -15,6 +16,8 @@
 import java.time.Duration;
 import java.util.List;
 
+import static com.intellij.remoterobot.search.locators.Locators.byXpath;
```

Comment:
The method that is using this Xpath needs to go in the UIBotTestUtils. Otherwise, it will be really confusing to have UIBot related code everywhere. The TestUtils class is mainly for non-bot generic functions.

Reply:
i added a method to ProjectFixture to get the popup window as a ContainerFixture - removed this xpath code form this class